website/docs/r/wafv2_web_acl_logging_configuration.html.markdown

```diff
@@ -73,6 +79,51 @@ resource "aws_wafv2_web_acl_logging_configuration" "example" {
 }
 ```
 
+## With Cloudwatch Log group and managed CloudWatch Log Resource Policy (See Notes)
```

Comment:
- Removed the `(See Notes)` so that we can hyperlink to this example. 
- Corrected title level to match other examples. 
- Adjusted capitalization in a couple of places.

```suggestion
### With CloudWatch Log Group and managed CloudWatch Log Resource Policy
```